src/cascadia/TerminalCore/TerminalSelection.cpp

```diff
@@ -63,6 +63,19 @@ std::vector<SMALL_RECT> Terminal::_GetSelectionRects() const
             selectionRow.Right = (row == lowerCoord.Y) ? lowerCoord.X : bufferSize.RightInclusive();
         }
 
+        // expand selection for Double/Triple Click
+        if (multiClickSelectionMode == selectionExpansionMode::Word && _selectionAnchor != _endSelectionPosition)

```

Comment:
so, this little section here: ` _selectionAnchor != _endSelectionPosition` is something that I can call a function in in #2152 

I'll be sure to clean it up if (a) that PR goes in first or (b) after I do a general cleanup to address #1327